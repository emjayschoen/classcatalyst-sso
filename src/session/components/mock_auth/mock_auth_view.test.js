import React from 'react';
import { shallow } from 'enzyme';
import MockAuthView from './mock_auth_view';

describe('<MockAuthView />', () => {
  let subject;
  beforeEach(() => {
    subject = shallow(<MockAuthView match={{ params: { provider: 'sid', returnTo: encodeURIComponent('/fun/and/profit') } }} />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
