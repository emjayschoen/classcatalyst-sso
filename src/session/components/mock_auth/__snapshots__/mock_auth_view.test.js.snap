// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MockAuthView /> renders correctly 1`] = `
<styled.div>
  <h3>
    This is not sid.
  </h3>
  <p>
    Sorry to disappoint you, but this is just a simulation...
  </p>
  <Link
    to="/finish_signin?returnTo=%2Ffun%2Fand%2Fprofit"
  >
    Finish signing in
  </Link>
</styled.div>
`;
